checks.d/mongo.py

```diff
@@ -6,124 +6,132 @@
+        "extra_info.page_faults": RATE,
+        "globalLock.activeClients.readers": GAUGE,
+        "globalLock.activeClients.total": GAUGE,
+        "globalLock.activeClients.writers": GAUGE,
+        "globalLock.currentQueue.readers": GAUGE,
+        "globalLock.currentQueue.total": GAUGE,
+        "globalLock.currentQueue.writers": GAUGE,
+        "globalLock.totalTime": GAUGE,
+        "mem.bits": GAUGE,
+        "mem.mapped": GAUGE,
+        "mem.mappedWithJournal": GAUGE,
+        "mem.resident": GAUGE,
+        "mem.virtual": GAUGE,
+        "metrics.document.deleted": RATE,
+        "metrics.document.inserted": RATE,
+        "metrics.document.returned": RATE,
+        "metrics.document.updated": RATE,
+        "metrics.getLastError.wtime.num": RATE,
+        "metrics.getLastError.wtime.totalMillis": RATE,
+        "metrics.getLastError.wtimeouts": RATE,
+        "metrics.operation.fastmod": RATE,
+        "metrics.operation.idhack": RATE,
+        "metrics.operation.scanAndOrder": RATE,
+        "metrics.queryExecutor.scanned": RATE,
+        "metrics.record.moves": RATE,
+        "metrics.repl.apply.batches.num": RATE,
+        "metrics.repl.apply.batches.totalMillis": RATE,
+        "metrics.repl.apply.ops": RATE,
+        "metrics.repl.buffer.count": GAUGE,
+        "metrics.repl.buffer.maxSizeBytes": GAUGE,
+        "metrics.repl.buffer.sizeBytes": GAUGE,
+        "metrics.repl.network.bytes": RATE,
+        "metrics.repl.network.getmores.num": RATE,
+        "metrics.repl.network.getmores.totalMillis": RATE,
+        "metrics.repl.network.ops": RATE,
+        "metrics.repl.network.readersCreated": RATE,
+        "metrics.repl.oplog.insert.num": RATE,
+        "metrics.repl.oplog.insert.totalMillis": RATE,
+        "metrics.repl.oplog.insertBytes": RATE,
+        "metrics.repl.preload.indexes.num": RATE,
+        "metrics.repl.preload.indexes.totalMillis": RATE,
+        "metrics.ttl.deletedDocuments": RATE,
+        "metrics.ttl.passes": RATE,
+        "opcounters.command": RATE,
+        "opcounters.delete": RATE,
+        "opcounters.getmore": RATE,
+        "opcounters.insert": RATE,
+        "opcounters.query": RATE,
+        "opcounters.update": RATE,
+        "opcountersRepl.command": RATE,
+        "opcountersRepl.delete": RATE,
+        "opcountersRepl.getmore": RATE,
+        "opcountersRepl.insert": RATE,
+        "opcountersRepl.query": RATE,
+        "opcountersRepl.update": RATE,
+        "replSet.health": GAUGE,
+        "replSet.replicationLag": GAUGE,
+        "replSet.state": GAUGE,
+        "stats.avgObjSize": GAUGE,
+        "stats.collections": GAUGE,
+        "stats.dataSize": GAUGE,
+        "stats.fileSize": GAUGE,
+        "stats.indexes": GAUGE,
+        "stats.indexSize": GAUGE,
+        "stats.nsSizeMB": GAUGE,
+        "stats.numExtents": GAUGE,
+        "stats.objects": GAUGE,
+        "stats.storageSize": GAUGE,
+        "uptime": GAUGE,
+    }
+
+    V2_ONLY_METRICS = {
```

Comment:
I just ran some tests against a MongoDB 2.6 and 3.0 instances, the `V2_ONLY_METRICS ` seens exhaustive.
`cursors.totalOpen` or `cursors.timedOut` are still part of MongoDB 3.0 as stated in the [MongoDB 3.0 documentation] (https://docs.mongodb.org/manual/reference/command/serverStatus/#serverStatus.cursors.totalOpen)